projects.json

```diff
@@ -5231,19 +5231,12 @@
         "action": "BuildSwiftPackage",
         "configuration": "release",
         "build_tests": "true",
-        "tags": "swiftpm",
-        "xfail": [
-          {
-            "issue": "https://github.com/swiftlang/swift/issues/75499",
-            "compatibility": ["5.10"],
-            "branch": ["main", "release/6.0"],
-            "platform": "Linux",
-            "job": ["source-compat"]
```

Comment:
This issue is actually not the exact issue Penny was having, but i can see why it's linked. The issue Penny was having was both related to a type with `Expression` in its name, as well as about type ambiguity. But It's still not quite what's in that issue.

Anyway, since I enabled the `build_tests_release`, I thought i'd need to remove the 5.10 version from `compatibility` array since I only recently added release-build testing support to Penny (we have CI for it too).

Reply:
The ambiguity problem was quite weird as far as i can remember. When i noticed it, i wasn't sure how the compiler figured out what type to infer in the first place. The type shouldn't have been visible by that file that eventually started throwing errors about ambiguity.